javascript/javascript3/week1/weather/weather.js

```diff
@@ -0,0 +1,61 @@
+const button = document.getElementById("button1");
+const input = document.getElementById("userinput");
+
+let body = document.querySelector("body");
+let ul = document.querySelector("ul");
+
+//const apiKey = "c65dee63b1543ccd79345b5fdaa81ee6";
+
+const url = `https://api.openweathermap.org/data/2.5/weather?q=copenhagen&appid=c65dee63b1543ccd79345b5fdaa81ee6&units=metric`;
+
+function inputLength() {
+  return input.value.length;
+}
+function createListElements() {
+  fetch(url)
+    .then((response) => response.json())
+    .then((data) => {
+      let h2 = document.createElement("h2");
+      h2.appendChild(document.createTextNode(data.name + "_" + "Weather"));
+      body.appendChild(h2);
+      const temp = document.getElementById("temperature");
+      temp.appendChild(
+        document.createTextNode(Math.floor(data.main.temp) + "-" + "kelvin")
```

Comment:
what can go wrong here ? if tmp will be null. check for empty or null value.

Reply:
i don't understand, we have an input for the city name, how could i make the temp value null. but without a value in a search bar , nothing will be displayed.